test/apis/v1alpha1/api_validation_integration_test.go

```diff
@@ -0,0 +1,809 @@
+package v1alpha1
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+
+	"k8s.io/utils/ptr"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1alpha1 "go.goms.io/fleet-networking/api/v1alpha1"
+)
+
+var _ = Describe("Test cluster v1 API validation", func() {
+	Context("Test MemberClusterService API validation - invalid cases", func() {
```

Comment:
```suggestion
	Context("Test MultiClusterService API validation - invalid cases", func() {
```

The description are wrong. Could you please self review first?